R/utilities.r

```diff
@@ -475,3 +475,6 @@ parse_safe <- function(text) {
   }
   out
 }
+
+# For older version of R
+isFALSE <- function(x) is.logical(x) && length(x) == 1L && !is.na(x) && !x
```

Comment:
Should this be in [backports.R](https://github.com/tidyverse/ggplot2/blob/master/R/backports.R) and have a version check?

Reply:
Sure, it seems the right place.